playground/worker/vite.config.ts

```diff
@@ -0,0 +1,17 @@
+import { fileURLToPath } from 'node:url';
+import { cloudflare } from '@flarelabs-net/vite-plugin-cloudflare';
+import { defineConfig } from 'vite';
+
+export default defineConfig({
+	plugins: [
+		cloudflare({
+			workers: {
+				worker: {
+					main: fileURLToPath(new URL('./src/index.ts', import.meta.url)),
```

Comment:
I think the relative path can be used directly here:

```suggestion
					main: './src/index.ts',
```

Reply:
good shout, thanks 🙂👍